src/option.h

```diff
@@ -478,19 +478,56 @@ namespace librealsense
         std::function<void(const option&)> _recording_function = [](const option&) {};
     };
 
+    class proxy_option : public option
+    {
+    public:
+        const char* get_value_description(float val) const override
+        {
+            return _proxy->get_value_description(val);
+        }
+        const char* get_description() const override
+        {
+            return _proxy->get_description();
+        }
+        virtual void set(float value) {}
```

Comment:
Leave this `= 0`

Reply:
Actually, my bad: since this is a "proxy option", it should by default simply foward ALL calls to _proxy -- including `set`